native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/internal/NativeImageCommandLineProvider.java

```diff
@@ -67,10 +67,11 @@ public class NativeImageCommandLineProvider implements CommandLineArgumentProvid
 
     private final Provider<NativeImageOptions> options;
     private final Provider<String> executableName;
-    private final Provider<String> outputDirectory;
     private final Provider<String> workingDirectory;
+    private final Provider<String> outputDirectory;
     private final Provider<RegularFile> classpathJar;
     private final Provider<Boolean> useArgFile;
+    private int majorJDKVersion = -1;
```

Comment:
:x: This should be a final field. Even if it's an API breaking change (the class is used in the Micronaut plugin).